numpy/core/tests/test_datetime.py

```diff
@@ -209,6 +208,26 @@ def test_datetime_scalar_construction(self):
         assert_equal(np.datetime64('1920-03-13T18:33:12.5', 'Y'),
                      np.datetime64('1920'))
 
+        # Construction from numpy types
+        assert_array_equal(np.datetime64(np.int(1262304000), 's'),
+                     np.datetime64(1262304000, 's'))
```

Comment:
`assert_equal` doesn't work, I'm guessing?

Reply:
Yes, it doesn't (#10081)